src/AspNet.Security.OAuth.Validation/Events/AuthenticationFailedContext.cs

```diff
@@ -2,7 +2,7 @@
 using Microsoft.AspNetCore.Http;
 using System;
 
-namespace AspNet.Security.OAuth.Validation.Events
```

Comment:
Well, we can follow that then. And I guess usage of the events is a bit easier without `.Events` suffix.